src/engine/enginedelay.cpp

```diff
@@ -24,12 +24,12 @@
 const int kiMaxDelay = 40000; // 208 ms @ 96 kb/s
```

Comment:
We cannot just pass the buffer size to the EngineDelay constructor because the EngineDelays are created in the EngineMaster constructor, which is run before SoundManager::setupDevices loads the configured buffer size.